plugins/tflint/main.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    null        = "~> 3.2.1"
+  }
+}
+
+resource "null_resource" "null_resource" {
+}
```

Comment:
why is this one different from the others?
if it's required by tflint, make all of them "tflinted"

Reply:
Done: 7836e0fa00583f059b551633c1ab8f4ca0e7428a